airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -380,7 +380,15 @@ def request_params(self, stream_slice: Mapping[str, Any] = None, **kwargs):
         params["expand[]"] = ["data.discounts", "data.taxes"]
         return params
 
+    @property
```

Comment:
Can we handle the response errors for all streams? It looks like the error could be floating.